core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/lru/LruBlockCacheConfiguration.java

```diff
@@ -114,7 +114,7 @@ public LruBlockCacheConfiguration(Configuration conf, CacheType type) {
 
     if (this.getSingleFactor() + this.getMultiFactor() + this.getMemoryFactor() != 1) {
       throw new IllegalArgumentException(
-          "Single, multi, and memory factors " + " should total 1.0");
+          "Single, multi, and memory factors  should total 1.0");
```

Comment:
Looks like this one revealed an extra space:
```suggestion
          "Single, multi, and memory factors should total 1.0");
```